src/js/lib/content/src/docsStyle.less

```diff
@@ -0,0 +1,325 @@
+@import "https://fonts.googleapis.com/css2?family=Source+Code+Pro:wght@300;400;500;600;700&display=swap";
+
+.doc {
+  font-family: DejaVuSansMonoBook, sans-serif;
+  .root {
+    display: flex;
+    margin-top: 4rem;
+  }
+  .main {
+    max-width: 650px;
+    min-width: 0;
+    margin-top: -20px;
+  }
+  .doc-code-container {
+    font-family: Source Code Pro, monospace;
+    font-size: 14px;
+    background-color: #e7ebef;
+    border-radius: 16px;
+    padding: 1px 5px 2px;
+    box-decoration-break: clone;
+    margin-top: 10px;
+    -webkit-box-decoration-break: clone;
+  }
+  code {
+    font-family: Source Code Pro, monospace;
+    font-size: 14px;
+    background-color: #e7ebef;
+    border-radius: 16px;
+    padding: 1px 5px 2px;
+    box-decoration-break: clone;
+    margin-top: 10px;
+    -webkit-box-decoration-break: clone;
```

Comment:
I still don't see here things prefixed with "enso." as we described - where is it done?

Reply:
It is - at top level, everything is in `.enso.docs {}`.